Blog(layout): Intrinsic layout overview

robdodson opened the pull request:
This splits #936 into a single PR for the first post and trims the text.

Meggin commented on src/site/content/en/blog/intrinsic-layout-overview/index.md:
```diff
@@ -0,0 +1,151 @@
+</blockquote>
+
+## Why "intrinsic"? 🤔
+This may look familiar, it's the result of an _extrinsic_ approach to layout:
```
Small nit. Change comma to semi-colon.

Meggin commented on src/site/content/en/blog/intrinsic-layout-overview/index.md:
```diff
@@ -0,0 +1,151 @@
+- The container doesn't handle overflow gracefully
+
+In my experience, initially laying out a site extrinsicly is the easiest. Later,
+it becomes tedious to maintain or even becomes a problem for other folks on the
```
Suggestion to change 'a problem for other folks on the team that aren't engineers' to 'a problem for team members who try and work on the site later on'.  Just a little wary of the 'not engineer part'.

Meggin commented on src/site/content/en/blog/intrinsic-layout-overview/index.md:
```diff
@@ -0,0 +1,151 @@
+
+In my experience, initially laying out a site extrinsicly is the easiest. Later,
+it becomes tedious to maintain or even becomes a problem for other folks on the
+team that aren't engineers.Perhaps you or your team have felt one or more of
```
+1 one to this suggestion-- I see the same missing space.

robdodson commented:
@argyleink do you have a video version of that gif? I was curious if the video might be smaller in file size.

argyleink commented:
i don't no.
good candidate for `loading="lazy"` though?

robdodson commented:
I've converted the gifs to videos, and added a blurb for the chaos one adam mentioned. Moved the post to `_drafts` so we can land the subsequent posts and roll the series out with a good cadence.
